docs/.vuepress/theme/global-components/IntegrationLinks.vue

```diff
@@ -0,0 +1,23 @@
+      </template>
+      <template #title>React</template>
+      <template #description>
+        Consume your API with React
+      </template>
+    </InstallLink>
+  </div>
+</template>
+
+<script>
+import InstallLink from "./InstallLink.vue"
```

Comment:
I think we need to add a bit more content to this page as it feels very empty right now, not entirely sure what type of content

Reply:
You are right! Let me fill this page